py3-rouge-score.yaml

```diff
@@ -29,10 +41,22 @@ pipeline:
       expected-sha256: c7d4da2683e68c9abf0135ef915d63a46643666f848e558a1b9f7ead17ff0f04
       uri: https://files.pythonhosted.org/packages/source/r/rouge_score/rouge_score-${{package.version}}.tar.gz
 
-  - name: Python Build
-    uses: python/build-wheel
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
 
-  - uses: strip
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
```

Comment:
remove py3-supported block in general do not add this subpackage.

Reply:
Don't we need to to provide for packages that will depend on this package once they're also building for multiple versions?